akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/ParameterDirectivesSpec.scala

```diff
@@ -49,15 +49,6 @@ class ParameterDirectivesSpec extends FreeSpec with GenericRoutingSpec with Insi
           }
         }
       }
-      "cause a MalformedRequestContentRejection on invalid query strings" in {
```

Comment:
The test should be adapted and kept (or is there another reason for deleting it?)